sdk/deviceupdate/azure-iot-deviceupdate/README.md

```diff
@@ -0,0 +1,67 @@
+# Device Update for IoT Hub client library for Java
+
+The library provides access to the Device Update for IoT Hub service that enables customers to publish updates for their IoT devices to the cloud, and then deploy these updates to their devices (approve updates to groups of devices managed and provisioned in IoT Hub). 
+
+  [Source code](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/deviceupdate)
+
+## Getting started
+
+The complete Microsoft Azure SDK can be downloaded from the [Microsoft Azure Downloads](https://azure.microsoft.com/en-us/downloads/?sdk=net) page and ships with support for building deployment packages, integrating with tooling, rich command line tooling, and more.
+
+For the best development experience, developers should use the official Microsoft NuGet packages for libraries. NuGet packages are regularly updated with new functionality and hotfixes.
+
+### Prerequisites
+
+- Microsoft Azure Subscription: To call Microsoft Azure services, you need to create an [Azure subscription](https://azure.microsoft.com/free/)
+- Device Update for IoT Hub instance
+- Azure IoT Hub instance
+
+### Authenticate the client
+
+In order to interact with the Device Update for IoT Hub service, you will need to create an instance of a [TokenCredential class](https://docs.microsoft.com/en-us/dotnet/api/azure.core.tokencredential?view=azure-dotnet) and pass it to the constructor of your AzureDeviceUpdateClientBuilder class.
+
+## Key concepts
+
+Device Update for IoT Hub is a managed service that enables you to deploy over-the-air updates for your IoT devices. The client library has three main components:
+
+- **Updates**: update management (import, enumerate, delete, etc.)
+- **Devices**: device management (enumerate devices and retrieve device properties)
+- **Deployments**: deployment management (start and monitor update deployments to a set of devices)
+
+You can learn more about Device Update for IoT Hub by visiting [Device Update for IoT Hub](https://github.com/Azure/adu-private-preview/tree/release/v0.2.0-private-preview).
+
+## Examples
+
+You can familiarize yourself with different APIs using [Samples](./src/samples).
+
+## Troubleshooting
+
+All Device Update for IoT Hub service operations will throw a ErrorResponseException on failure with helpful ErrorCodes.
+
+For example, if you use the `getUpdateAsync` operation and the model you are looking for doesn't exist, you can catch that specific [HttpStatusCode](https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netcore-3.1) to decide the operation that follows in that case.
```

Comment:
Fix the link - it is pointing to .NET documentation.

Reply:
I don't see a Java version anywhere. I can only see C#, VB, C++ and F#!